crypto/mem.c

```diff
@@ -290,72 +209,4 @@ void CRYPTO_clear_free(void *str, size_t num, const char *file, int line)
         OPENSSL_cleanse(str, num);
     CRYPTO_free(str, file, line);
 }
-
-#if !defined(OPENSSL_NO_CRYPTO_MDEBUG)
-
-# ifndef OPENSSL_NO_DEPRECATED_3_0
```

Comment:
I agree that for now removing Only pre-3.0 deprecations is the right approach. These functions were only deprecated in 3.0, though.

Reply:
agreed, do not remove things deprecated in 3.0 or later.